### ISSUE TITLE
Accessibility features violated if options are changed on calendar click

### ISSUE THREAD

**Issue by simakas** [kekeh/mydatepicker#583]
how to reproduce:
- dateChanged -> onDateChanged
- change highlightDates array in myDatePickerOptions (using copy or any other method)
- calendar gets re-rendered
- tab does not work

suggested fix:
- add directive for selectedday span (preferred) to survive re-rendering
